force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +102,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        if(a == null){
+            return null;
+        } else if (math.mod(a, 2) == 0){
+            return 'Even';
+        } else
```

Comment:
Although not wrong,  curly braces are encouraged specially if you need to add code later.
They are a standard practice and helps with readability